default.nix

```diff
@@ -0,0 +1,59 @@
+let
+  holonixPath = builtins.fetchTarball { # main as of Mar 15, 2022
+    url = "https://github.com/holochain/holonix/archive/391557dc5b3065b0d357ea9f9a2bc77e7347be8e.tar.gz";
```

Comment:
@weswalla try going to holonix and finding a most recent commit. 
reference it in the above URL in place of `391...`

this may allow for the inclusion of rust 1.63